lib/api/CDataFrameTrainBoostedTreeClassifierRunner.cc

```diff
@@ -216,14 +204,29 @@ void CDataFrameTrainBoostedTreeClassifierRunner::writePredictedCategoryValue(
     }
 }
 
+CDataFrameTrainBoostedTreeClassifierRunner::TLossFunctionUPtr
+CDataFrameTrainBoostedTreeClassifierRunner::loss(std::size_t numberClasses) {
+    using namespace maths::boosted_tree;
```

Comment:
Is this needed?

Reply:
I use `CBinomialLogisticLoss` and `CMultinomialLogisticLoss` unqualified in the next line. I found with auto formatting this made the code a bit more readable.